### PR TITLE
FAQ revamp and typo fixes

### PR DESCRIPTION
**Pull request by kn0wmad**
Finished reading through all the docs, fixed some typos, and added some f-droid links alongside play store.  Will plan much more intensive work on docs improvements next :)

### CODE REVIEW

**Comment by kn0wmad** [comment]
Complete FAQ revamp, with Concepts edits.  Please proofread

File: source/misc-guides/tor-os/android.rst [hunk 1]
```diff
@@ -4,7 +4,7 @@
 Running Tor on Android
 **********************
 
-First, install the Tor proxy service to your system. To do so, open the play store and download `Orbot <https://play.google.com/store/apps/details?id=org.torproject.android>`_. Once you have done so, launch Orbot.
+First, install the Tor proxy service to your system. To do so, download Orbot from the `Play Store <https://play.google.com/store/apps/details?id=org.torproject.android>`_ , or from `F-Droid <https://f-droid.org/packages/org.torproject.android>`_ . Once you have done so, launch Orbot.
```
Comment (elvece) [inline source/misc-guides/tor-os/android.rst:7]: 👍

File: source/support/faq.rst [hunk 2]
```diff
@@ -4,104 +4,107 @@
...
-
-When you see your credit card information on your banking app, your messages in your texting app, your passwords in your password management app, all of that information comes from somewhere in the cloud: some server run by some company somewhere on the planet. Who can see the data stored in that server? Who can edit it? It's not always clear, but the increasingly common practice of selling your data to advertisers and the high-profile cyber-security breaches of the last decade suggest a pessimistic outlook.
+What is the Embassy?
+--------------------
+The Embassy is a "shelf-top" computer built using a Raspberry Pi (relevant link) for hardware and running EmbassyOS software.
 
-One thing is for certain though: if you control your server, then you control your data. Your finances, your communications, all of it is actually yours -- and only yours -- with an Embassy.
+What is EmbassyOS?
+------------------
+EmbassyOS is a new kind of Operating System (OS).  It is built from the ground up starting with a stripped-down version of Raspbian Buster Lite (relevant link).  It handles all operations of the device, including managing the Marketplace, Services, and Service data.
```
Comment (elvece) [inline source/support/faq.rst]: yes this should link to the embassy os concept, i think its like `:ref:`embassy`` or `:ref:`custom text <embassy>``

File: source/support/faq.rst [hunk 3]
```diff
@@ -4,104 +4,107 @@
...
 
+What are EmbassyOS Services?
+----------------------------
+A Service can be any piece of software added to the Marketplace.  All services are "self-hosted," (relevant link?) meaning that you are in complete control of your data.  This means you can run your own "cloud!"
```
Comment (elvece) [inline source/support/faq.rst]: maybe link to managing or available services?

**Review by elvece — APPROVED** [review]

File: source/support/faq.rst (hunk 2 above)
Comment (kn0wmad) [reply source/support/faq.rst]: Done

File: source/support/faq.rst (hunk 3 above)
Comment (kn0wmad) [reply source/support/faq.rst]: Linked to both.  Thank you for the feedback :)

File: source/support/faq.rst [hunk 4]
```diff
@@ -4,104 +4,112 @@
...
 
-In other words, it is a generalized private personal server capable of running all sorts of self hosted open source software. 
+What is Start9Labs?
+-------------------
+Start9Labs is a company based in Denver, CO that builds the Embassy and EmbassyOS.
 
-When you see your credit card information on your banking app, your messages in your texting app, your passwords in your password management app, all of that information comes from somewhere in the cloud: some server run by some company somewhere on the planet. Who can see the data stored in that server? Who can edit it? It's not always clear, but the increasingly common practice of selling your data to advertisers and the high-profile cyber-security breaches of the last decade suggest a pessimistic outlook.
+What is the Embassy?
+--------------------
+The Embassy is a "shelf-top" computer built using a `Raspberry Pi <https://www.raspberrypi.org/products/raspberry-pi-4-model-b/>`_ for hardware and running EmbassyOS software.
```
[the remark this answers is not in the document]
Comment (kn0wmad) [reply source/support/faq.rst:17]: Linking to the Embassy under Concepts for an expanded answer

File: source/support/faq.rst [hunk 5]
```diff
@@ -4,104 +4,112 @@
...
-I cannot connect to a service.
-==============================
+What if I can't connect to my Embassy?
+--------------------------------------
+Via WiFi - Please ensure your phone / computer is connected to the same wireless network as your Embassy.
 
-Make sure the service is started by viewing it in the Services tab in the Embassy dashboard menu. A green indicator bar should be visible.
+Via Ethernet - Please ensure your phone / computer is connected to the same network as the router.  Be careful that you are not on a seperate or "guest" network.
 
+Why does the Bitcoin service take so long to be ready?
```
[the remark this answers is not in the document]
Comment (kn0wmad) [reply source/support/faq.rst:101]: Moved

File: source/support/faq.rst [hunk 6]
```diff
@@ -4,104 +4,112 @@
...
 
+How do I know if my Embassy is running?
+---------------------------------------
+After plugging into power and internet, you will hear 2 distinct sounds: first, a “bep” ‐ indicating the device is powering on, and second, a “chime” ‐ indicating the device is ready for setup.
 
-I cannot connect to a service.
-==============================
+What if I can't connect to my Embassy?
+--------------------------------------
+Via WiFi - Please ensure your phone / computer is connected to the same wireless network as your Embassy.
```
[the remark this answers is not in the document]
Comment (kn0wmad) [reply source/support/faq.rst:97]: Combined

File: source/support/faq.rst [hunk 7]
```diff
@@ -4,104 +4,112 @@
...
+What is the Embassy?
+--------------------
+The Embassy is a "shelf-top" computer built using a `Raspberry Pi <https://www.raspberrypi.org/products/raspberry-pi-4-model-b/>`_ for hardware and running EmbassyOS software.
 
-One thing is for certain though: if you control your server, then you control your data. Your finances, your communications, all of it is actually yours -- and only yours -- with an Embassy.
+What is EmbassyOS?
+------------------
+EmbassyOS is a new kind of Operating System (OS).  It is built from the ground up starting with a stripped-down version of `Raspbian Buster Lite <https://www.raspbian.org/>`_.  It handles all operations of the device, including managing the Marketplace, Services, and Service data.  Learn more :ref:`here <embassyos>`.
```
[the remark this answers is not in the document]
Comment (kn0wmad) [reply source/support/faq.rst]: Changed, let me know if it can still be more exciting.  I'm going to look for additional FAQs in the Telegram now.  Thanks

**Pull request merged**